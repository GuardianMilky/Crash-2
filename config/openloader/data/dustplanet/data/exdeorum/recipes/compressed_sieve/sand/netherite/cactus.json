{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/sand"},"mesh":"exdeorum:netherite_mesh","result":"minecraft:cactus","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.15}}