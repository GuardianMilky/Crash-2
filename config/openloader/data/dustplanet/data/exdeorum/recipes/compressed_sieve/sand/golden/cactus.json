{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/sand"},"mesh":"exdeorum:golden_mesh","result":"minecraft:cactus","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.1}}